helm/ph-ee-engine/connector-airtel/values.yaml

```diff
@@ -0,0 +1,70 @@
+service:
+  apiversion: "v1"
+
+secret:
+  apiversion: "v1"
+
+configmap:
+  apiversion: "v1"
+
+
+enabled: false
+image: ""
+imageTag: latest
+springProfileActive: ""
+loggingLevelRoot: "INFO"
+zeebeInitTransferWaitTimer: 10
+tenant: "gorilla"
+callbackHost: ""
+retryCount: 3
+apiTimeout: 60000
+channel:
+  host: "http://ph-ee-connector-channel"
+amsMifos:
+  host: "http://ph-ee-connector-ams-mifos"
+accounts:
+  amsMifos:
+    name: "ams_mifos"
+    businessShortCode: ""
+    till: ""
+    authHost: "https://fynams.sandbox.mifos.io/"
+    apiHost: "https://fynams.sandbox.mifos.io/"
+paybill:
+  accountHoldingInstitutionId: "gorilla"
+  amsMifos:
+    businessShortCode: ""
+    currency: "KES"
+limits:
+  cpu: "500m"
+  memory: "512M"
+requests:
+  cpu: "100m"
+  memory: "256M"
+# Enabling this will publicly expose your Elasticsearch instance.
+# Only enable this if you have security enabled on your cluster
+ingress:
+  enabled: false
+  annotations: {}
+  # kubernetes.io/ingress.class: "nginx"
+  pathtype: ImplementationSpecific
+  hosts:
+    - host: ""
+      paths:
+        - path: /
+  tls: []
+  #  - secretName: sandbox-secret
+  #    hosts:
+  #      - chart-example.local
+
+deployment:
+  apiVersion: "apps/v1"
+  annotations:
+    deployTime: "{{ .Values.deployTime }}"
+skip:
```

Comment:
What is this configuration for?

Reply:
We might use Mpesa instead of Airtel based on the tenant. However, according to this [comment](https://github.com/fynarfin/ph-ee-connector-airtel/pull/1#discussion_r1670260251), such changes should be made in the channel connector. Therefore, these changes have been removed.